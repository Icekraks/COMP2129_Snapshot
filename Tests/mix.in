set abc 1 2 3 3 3
uniq abc
get abc
pop abc
push abc 1
append abc 1
get abc
list keys
list entries
max abc
pop abc
bye
