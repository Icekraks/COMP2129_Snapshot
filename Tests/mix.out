ok
[1 2 3 4 5]
1
ok
ok
[1 2 3 4 5 1]
a
a [1 2 3 4 5 1]
5
ok
bye
